packages/flutter_tools/templates/app_shared/ios-objc.tmpl/RunnerTests/RunnerTests.m.tmpl

```diff
@@ -3,6 +3,11 @@
 #import <XCTest/XCTest.h>
 {{#withPlatformChannelPluginHook}}
 
+{{#withSwiftPackageManager}}
+// If your plugin has been explicitly set to "type: .dynamic" in the Package.swift,
+// you will need to add your plugin as a dependency of RunnerTests within Xcode.
+{{/withSwiftPackageManager}}
```

Comment:
@stuartmorgan This is where you were thinking I include the message, right?

Reply:
Yep, perfect!